wrapcheck/wrapcheck.go

```diff
@@ -273,6 +277,16 @@ func reportUnwrapped(
 	pkgGlobs []glob.Glob,
 ) {
 
+	if cfg.ReportInternalErrors {
+		// Check if the call is package-internal function
+		fnIdent, ok := call.Fun.(*ast.Ident)
+		if ok {
+			fnSig := pass.TypesInfo.ObjectOf(fnIdent).String()
+			pass.Reportf(tokenPos, "error should be unwrapped: sig: %s", fnSig)
```

Comment:
I think the error message here should be more along the lines of `error should be wrapped` as the purpose of this is to identify incorrectly unwrapped errors.

```suggestion
			pass.Reportf(tokenPos, "package-internal error should be wrapped: sig: %s", fnSig)
```

Reply:
Thank you for your comment.
I have fixed the error messages.